### PR TITLE
Nuke: LoadClip colorspace override - OP-6591

### PR DESCRIPTION
**Pull request by tokejepsen**
## Changelog Description
Setting the colorspace from the representation data was not supported.

## Testing notes:
1. Setup colormanagement for Nuke.
2. Publish write instance with modified colorspace (`color_picking` for example).
3. Load publish write instance and validate the correct colorspace is set.
4. Add global or host overrided File rules with some other colorspace on your project
5. Load again and notice that the colorspace was overriden (priority on filerules should override colorspaceData)

### CODE REVIEW

**Comment by tokejepsen** [comment]
@jakubjezek001 ready for testing now.

File: openpype/hosts/nuke/plugins/load/load_clip.py [hunk 1]
```diff
@@ -140,6 +142,7 @@ def load(self, context, name, namespace, options):
                 "Representation id `{}` is failing to load".format(repre_id))
             return
 
+        print(representation["data"]["colorspaceData"])
```
Comment (tokejepsen) [inline openpype/hosts/nuke/plugins/load/load_clip.py:145]:
```suggestion
```

File: openpype/hosts/nuke/api/utils.py [hunk 2]
```diff
@@ -117,19 +118,22 @@ def get_colorspace_list(colorspace_knob):
...
+    # This pattern is to match with roles which uses an indentation and
+    # parentheses with original colorspace. The value returned from the
+    # colorspace is the string before the indentation, so we'll need to
+    # convert the values to match with value returned from the knob,
+    # ei. knob.value().
+    pattern = r".*\t.* \(.*\)"
+    for colorspace in nuke.getColorspaceList(colorspace_knob):
+        match = re.search(pattern, colorspace)
+        if match:
+            results.append(colorspace.split("\t")[0])
```
Comment (tokejepsen) [inline openpype/hosts/nuke/api/utils.py:132]:
```suggestion
            results.append(colorspace.split("\t", 1)[0])
```

**Review by jakubjezek001 — APPROVED** [review]
tested and worked well

**Pull request merged**